cmake/sysbuild/fast_pair_hex.cmake

```diff
@@ -143,6 +143,10 @@ function(fast_pair_hex_dts)
   )
 endfunction()
 
+if(NOT DEFINED FP_MODEL_ID OR NOT DEFINED FP_ANTI_SPOOFING_KEY)
+  message(FATAL_ERROR "Automatic Fast Pair provisioning data generation is enabled but `FP_MODEL_ID` or `FP_ANTI_SPOOFING_KEY` were not provided.")
+endif()
```

Comment:
revert back to how it was with error in .c file so this doesn't completely break someone's build

Reply:
The point is that the **error** is a c build time error, not a cmake configuration time error, a cmake configuration error nukes a whole project completely annoying the person as they have to clear it and restart